pkgs/top-level/perl-packages.nix

```diff
@@ -16332,6 +16332,12 @@ let
       url = mirror://cpan/authors/id/M/MZ/MZSANFORD/Sys-CPU-0.61.tar.gz;
       sha256 = "1r6976bs86j7zp51m5vh42xlyah951jgdlkimv202413kjvqc2i5";
     };
+    patches = [
+      # Bug #95400 for Sys-CPU: Tests fail on ARM and AArch64 Linux
+      # https://rt.cpan.org/Public/Bug/Display.html?id=95400
+      ../development/perl-modules/perl-SysCPU-Add-support-for-cpu_type-on-ARM-and-AArch64-Linux-pl.patch
+      ../development/perl-modules/perl-SysCPU-cpu_clock-can-be-undefined-on-an-ARM.patch
```

Comment:
Same:

https://rt.cpan.org/Ticket/Attachment/1388036/737125/0002-cpu_clock-can-be-undefined-on-an-ARM.patch